### ISSUE TITLE
ISC DHCP server 4.3.2 stopped issuing adress after RELEASE on Zyxel Keenetic

### ISSUE THREAD

**Issue by bredis** [Entware/entware#179]
I forgive about tion for my English - I use online translator.
Tested on white and black Zyxel Keenetic.
The bottom line is that after RELEASE server stops issuing addresses. He begins to give out only after the stop / start. This RENEW can make any number of times, but after the server thinks RELEASE.
I check this configuration on Debiane - everything works fine.

**Comment by ryzhovau** [comment]
Feel free to discuss it [here](http://forums.zyxmon.org/viewforum.php?f=5) (in Russian).

Add a new issue [here](https://github.com/Entware-ng/Entware-ng/issues) if necessary.
